doc/UsersGuide/BuildingRunningTesting/BuildSRW.rst

```diff
@@ -361,6 +350,42 @@ The build will take a few minutes to complete. When it starts, a random number i
 
    If you see the ``build.out`` file, but there is no ``ufs-srweather-app/exec`` directory, wait a few more minutes for the build to complete.
 
+.. _install-uw:
+
+Install ``uwtools``
+^^^^^^^^^^^^^^^^^^^^
+
+The :uw:`UW Tools documentation <sections/user_guide/installation.html>` has the most up-to-date installation instructions. Users should refer to that documentation as authoritative. The UW team welcomes questions in its :uw-repo:`GitHub Discussions <discussions>` forum. 
+
```

Comment:
Suggesting to add in the beginning of the next line (360):
```
When SRW App is built using ./devbuils.sh script, it also installs conda environment and uwtools.
```

Reply:
This section about installing `uwtools` is specific to the CMake Approach (it is a subsection). Users running with the devbuild.sh approach shouldn't make it to this section at all.